qt/test/test_lint.py

```diff
@@ -30,50 +110,114 @@ def _collect_py_files(self) -> Iterable[pathlib.Path]:
 
         if not path.name.startswith('test'):
             raise RuntimeError('Something went wrong. The test should run '
-                               'inside the test folder but current folder '
+                               'inside the test folder but the current folder '
                                f'is {path}.')
 
         # Workaround
         path = path.parent
 
         # Find recursive all py-files.
-        return path.rglob('**/*.py')
+        py_files = path.rglob('**/*.py')
```

Comment:
Same as above, this should be either:

```suggestion
        py_files = path.glob('**/*.py')
```

OR

```suggestion
        py_files = path.rglob('*.py')
```